bin/yarnnodelabeltest.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# This example requires a yarn cluster that contains some slave nodes that are labeled as "mylabel" and the others are not labeled.
+# REEFYarnNodeLabelTestDriver submits two EvaluatorRequests, one is labeled as "mylabel" and the other is not labeled.
+# This test passes if one container is allocated on the labeled node, and another one is on default node.
+
+export REEF_TEST_YARN=true
+DEPENDENCY_JAR=`echo $REEF_HOME/lang/java/reef-tests/target/reef-tests-*-test-jar-with-dependencies.jar`
+CLASSPATH=`yarn classpath`
+
+CMD="java -cp $YARN_CONF_DIR:$DEPENDENCY_JAR:$CLASSPATH org.junit.runner.JUnitCore org.apache.reef.tests.yarnnodelabel.NodeLabelTest $*" 
```

Comment:
@scan3 My point is that we should be able to run tests like
```bash
runyarntests.sh org.apache.reef.tests.yarnnodelabel.NodeLabelTest ...
```
instead of creating a new shell script just for one test suite.